packages/graphql-server/src/functions/graphql.ts

```diff
@@ -34,6 +34,7 @@ import { ValidationError } from '../errors'
 
 import type { GraphQLHandlerOptions } from './types'
 import { Headers, Request } from 'cross-undici-fetch'
+import isString from 'lodash/isString'
```

Comment:
Thanks for the fix.

Might we not use loadash here? 

We have to be cautious of including packages on the api side when deploying to Netlify and Vercel.

Reply:
Hi @tristanlee85 We'd like to get this in a patch today, could you revisit the use of `loads` here to limit the packages that then deployed? Thanks!

Reply:
My apologies, it auto-imported the fn so I assumed it was safe to use, but not necessary. I've now just wrapped it in `String(...)` just to ensure it is a string primitive. This is plenty as well.

Reply:
Thanks! Appreciate this fix!